src/atomic.rs

```diff
@@ -0,0 +1,171 @@
+use cid::multihash::Code::Blake2b256;
```

Comment:
Just one note, this file will change significantly after this PR is merged: https://github.com/consensus-shipyard/builtin-actors/pull/17

@sergefdrv, you should maybe consider closing that PR and working on this repo directly.